### PR TITLE
tracing: Elide redundant info in tracing contexts

### PR DESCRIPTION
**Pull request by olix0r**
The tracing context includes redundnant information, particularly about
the traffic target.

This change modifies the accept stack to include both the source peer and
target addresses in the `accept` context; and the target address has
been removed from intermediate contexts, many of which can be moved to
the debug level now.

### CODE REVIEW

File: linkerd/app/inbound/src/lib.rs [hunk 1]
```diff
@@ -354,13 +354,8 @@ impl Config {
                     .box_http_request()
                     .box_http_response(),
             )
+            .instrument(|_: &_| debug_span!("source",))
```
Comment (hawkw) [inline linkerd/app/inbound/src/lib.rs:357]: nit:
```suggestion
            .instrument(|_: &_| debug_span!("source"))
```

**Review by kleimkuhler — APPROVED** [review]
👍

File: linkerd/app/core/src/serve.rs [hunk 2]
```diff
@@ -36,6 +36,7 @@ where
                     let span = info_span!(
                         "accept",
                         peer.addr = %addrs.peer(),
+                        target.adr = %addrs.target_addr(),
```
Comment (hawkw) [inline linkerd/app/core/src/serve.rs:39]: typo:
```suggestion
                        target.addr = %addrs.target_addr(),
```

**Review by hawkw — COMMENTED** [review]
noticed a couple typos. LGTM otherwise!

**Review by hawkw — APPROVED** [review]

**Pull request merged**